### PR TITLE
Feat/ota 2871/user can edit odn

### PR DESCRIPTION
**Pull request by piotrcichonmee**
[OTA-2871] User can edit their ODN

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx [hunk 1]
```diff
@@ -150,6 +203,18 @@ class ProfileOrganization extends Component {
             </div>
           </span>
         </span>
+        {(menuEditShowMenu) && (
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx]: Unnecessary parentheses `(menuEditShowMenu)`

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 2]
```diff
@@ -53,11 +55,28 @@ export default class UserStore {
...
     resetAsync(this.contractsAcceptAsync);
+    resetAsync(this.editOrganizationNameAsync);
     resetAsync(this.getOrganizationsAsync);
     resetAsync(this.getOrganizationUsersAsync);
     resetAsync(this.addUserToOrganizationAsync);
   }
 
+  editOrganizationName = async (namespace, name) => {
+    const namespaceCookie = Cookies.get(ORGANIZATION_NAMESPACE_COOKIE);
+    // just to be sure user edits his won organization
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: `own` not `won`

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 3]
```diff
@@ -53,11 +55,28 @@ export default class UserStore {
...
+    if (namespace === namespaceCookie) {
```
Comment (houcros) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: What do we need this check at all? The `PATCH` endpoint will use the current endpoint. I would say that this `editOrganizationName` method should only take the new name as parameter.

Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: Yes, indeed. It was before the endpoint has changed. Done.

File: ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx]: Done

**Pull request merged**